src/message.ts

```diff
@@ -613,6 +625,40 @@ export class Message implements Sayable {
   // }
 
   /**
+   * @param sendTo UserId/RoomId/Room/Contact
+   */
+  public forward(room: Room): Promise<any>
+  public forward(contact: Contact): Promise<any>
+  public forward(id: string): Promise<any>
+  public forward(sendTo: string|Room|Contact): Promise<any> {
+    const m = <MsgRawObj>this.rawObj
```

Comment:
When I remove it, the typescript check will be error.

```
src/message.ts(645,35): error TS2532: Object is possibly 'undefined'.
src/message.ts(645,61): error TS2532: Object is possibly 'undefined'.
src/message.ts(646,28): error TS2532: Object is possibly 'undefined'.
src/message.ts(648,43): error TS2532: Object is possibly 'undefined'.
src/message.ts(650,9): error TS2532: Object is possibly 'undefined'.
src/message.ts(651,30): error TS2532: Object is possibly 'undefined'.
src/message.ts(652,9): error TS2532: Object is possibly 'undefined'.
src/message.ts(653,25): error TS2532: Object is possibly 'undefined'.
src/message.ts(654,9): error TS2532: Object is possibly 'undefined'.
src/message.ts(655,50): error TS2532: Object is possibly 'undefined'.
src/message.ts(658,16): error TS2345: Argument of type 'MsgRawObj | undefined' is not assignable to parameter of type 'MsgRawObj'.
  Type 'undefined' is not assignable to type 'MsgRawObj'
```

Perhaps, I should in the Message constructor, do the following changes?

```javsscript
  constructor(public rawObj?: MsgRawObj) {

  // this.rawObj = rawObj = rawObj || <MsgRawObj>{}
   this.rawObj = rawObj = rawObj as MsgRawObj || <MsgRawObj>{}

  }
```